website/content/docs/enterprise/audit-logging.mdx

```diff
@@ -8,9 +8,9 @@ description: >-
 # Audit Logging
 
 <EnterpriseAlert>
-  This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Governance and Policy module.
+  This feature is available in versions 1.8.0+ of
+  HashiCorp Cloud Platform (HCP) and self-managed Consul according to the{' '}
+  <a href="/docs/enterprise/feature-availability">enterprise feature matrix</a>.
```

Comment:
```suggestion
  This feature is requires version 1.8.0+ of
  HashiCorp Cloud Platform (HCP) or self-managed Consul. Refer to the {' '}
  <a href="/docs/enterprise/feature-availability">enterprise feature matrix</a> for additional information.
```
Just to reiterate, it would be better to specify in a 'Requirements' section.  
We can do that down the road. For now, I suggest rewording all of these instances to fix the odd cause/effect construction " . . . according to the enterprise feature matrix".

Reply:
Reworded accordingly. Moving to a requirements section can happen in a future PR.